tests/nosetests/pyanaconda_tests/module_storage_test.py

```diff
@@ -75,6 +76,14 @@ def setUp(self):
         self.storage_module = StorageModule()
         self.storage_interface = StorageInterface(self.storage_module)
 
+    def _test_dbus_property(self, *args, **kwargs):
```

Comment:
The same here. Please name it `check` instead of `test`.

Reply:
Fixed.